.github/workflows/cypress.yaml

```diff
@@ -0,0 +1,54 @@
+# Run the tests
+# Todo : reuse a build from another action, instead of rebuilding for nothing.
+# tchap modification : use "./scripts/layered.with.version.sh" instead of "./scripts/layered.sh"
+name: Cypress End to End Tests
+on:
+  pull_request: { }
+  push:
+    branches: [ develop, master ]
+  repository_dispatch: # :TCHAP: do we need this ?
```

Comment:
We could remove this as I don't think we will use this yet.
I think it was used by an external CI to trigger the workflow.

More informations on dispatch : 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch